sentry-android-integration-tests/test-app-plain/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "io.sentry.java.tests.perf.appplain"
+        minSdk 21
+        targetSdk 32
+        versionCode 1
```

Comment:
All this sort of configuration could come from the shared `Config` class.

Reply:
My idea was to keep those files close to the original template as possible (a.k.a use the defaults)